src/post_distribute.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/lambda/python:3.11
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+COPY dashboard/post_distribute/requirements.txt .
+RUN pip install -r requirements.txt
+COPY dashboard/post_distribute/post_distribute.py .
+COPY shared shared
+
+# Force setup of some initial matplotlib configuration artifacts
+RUN mkdir /tmp/matlplotlib
+ENV MPLCONFIGDIR=/tmp/matlplotlib
+RUN cumulus-library version
```

Comment:
This was stupid enough that it almost caused me to pull matplotlib from the library and move the PSM charting to a different library. I may still do this.

Otherwise - matplotlib freaks out over not having home access and tries to find a new place to put things, and this often blows our 10 second window for initializing the lambda.